### DISCUSSION TITLE
Out Of Date Status Bug?

### DISCUSSION THREAD

**Discussion by phunsanit** [chocolatey/home#242] [Q&A] [answered]
I update Visual studio 2022 by it self from version 117.4.3.0 to 117.4.4 ( current version ). 
but  chocolatey GUI tell me "Out Of Date" status.

**Comment by pauby** [comment]
You haven't provided any information, logs, steps to reproduce or screenshots. There isn't much we can do without that.

**Comment by TheCakeIsNaOH** [comment, marked as answer]
@phunsanit When you say "update Visual studio", do you mean using the Visual Studio Installer to update Visual Studio? 

If so, then this behavior is expected. Using the Visual Studio Installer will update the Visual Studio software, but the Chocolatey package for Visual Studio will still be on the old version (and be outdated). ChocolateyGUI (and Chocolatey CLI) displays outdated packages, not outdated software.

**Comment by gep13** [comment]
@phunsanit can I please suggest that we keep discussions about this topic in one place?  Posting in multiple places, as you have done here: http://disq.us/p/2stfyhu just causes more work for maintainers.  Let's keep the discussion here in GitHub for now.

**Reply by pauby** [reply]
I've gone ahead and deleted the Disqus comment, so we can keep everything here.
